start/scaling-up/scaling-up.md

```diff
@@ -0,0 +1,70 @@
+[title]: - "Scale Up your Workflow on OSG Connect"
+[TOC]
+
+## Background
+
+Much of OGS's computing power comes from the ability to run a large number 
+of jobs simulateously. Many HTC-friendly workflows involve running the same 
+job multiple times with no or minor changes between them. Here, we will 
+talk about how to submit multiple jobs at once using the `queue` command as 
+well as some important considerations and tips for scaling up your HTCondor jobs. 
+For a hands-on example of these concepts, checkout the [OSG Connect Quickstart]
+(https://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart). 
+
+## Things to Consider
+
+In order to avoid confusion, let's review some HTCondor terminiology:
+ - When you queue a single submit script using the `condor_submit` command, this is 
+called a **cluster** and can consist of one more individual jobs.
+ - Each individual job within a cluster is called a **process**. Therefore, each 
+*cluster* can be made up of one or more *processees*. Each proceess lands on its 
+own job slot and is handled individually from the other processes in the cluster.
+
+### Handling Output
+
+HTCondor has built in features that make submitting multiple jobs with the same 
+submit script possible and easy. However, this means that you will need to ensure 
+that the output created by each individual process will be uniquely identified so 

```

Comment:
```suggestion
that the output created by each individual job will be uniquely identified so 

```

Reply:
why use 'job' here and not 'process'?

Reply:
I guess either is fine. Process is an overloaded term that has other meanings, so I wanted to make it clear that we're talking about a job here, not a process within a larger program.

Reply:
I went with process to keep terms straight with the assumption that the reader would use the definition above and that using job would add confusion.

Reply:
I could italicize it to further emphasize that I'm using the same term as defined.

Reply:
I think it should be fine. You can leave it as is. :)